microservices/requestApi/routes/v2/routes/requests.js

```diff
@@ -39,23 +39,36 @@ var getRouter = function(db){
                 res.json({error: formErr});
                 return;
             }
-            res.json(JSON.parse(formRes));
+            var r = formRes
+            try{
+                r = JSON.parse(formRes);
+            }catch(ex){}
+
+            res.json(r);
```

Comment:
On some machines I've seen it be a string instead of JSON, by wrapping the conversion in a try catch and ignoring the exception we guarantee it is always an object before the next line.